lib/wraft_doc/document/content_type.ex

```diff
@@ -2,6 +2,8 @@ defmodule WraftDoc.Document.ContentType do
   @moduledoc """
     The content type model.
   """
+  @behaviour ExTypesense
```

Comment:
Why do we need this?

Reply:
it was in the Extypesense sample they provided